change/monosize-bundler-esbuild-53dd0c0b-96e9-4daa-9082-b333ff52c6cd.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

```suggestion
  "type": "minor",
```

these packages are `0.x`

---

aaah, we should configure our release tool to enforce this.